rails/config/routes.rb

```diff
@@ -292,25 +327,12 @@
     get '/report/learner' => 'report/learner#index', :as => :learner_report
     get '/report/learner/updated_at/:id' => 'report/learner#updated_at', :as => :learner_updated_at
     get '/report/learner/report_only' => 'report/learner#report_only', :as => :learner_report_only
+    get '/report/learner/update_learners' => 'report/learner#update_learners'
```

Comment:
This is a before_filter so it likely shouldn't be exposed.

Reply:
No, its an endpoint, it is referenced by the before filter though:

```
  before_filter :setup,
      :only => [
      :index,
      :update_learners
    ]
```

There is also an existing controller test for it.